src/languages/en.js

```diff
@@ -522,6 +522,8 @@ export default {
         tryAgain: 'Try again',
         verifyIdentity: 'Verify identity',
         genericError: 'There was an error while processing this step. Please try again.',
+        cameraRequestTitle: '"New Expensify" Would Like to Access the Camera',
+        cameraRequestMessage: 'Uh-oh, you have not granted us camera access. We need that to complete this verification',
```

Comment:
@phivh David from marketing team came back with the copy and he suggested one change. I have also asked internally for spanish translation of these and I will send them over here asap.
```suggestion
        cameraRequestMessage: 'Uh-oh, you have not granted us camera access. We need access to complete verification.',
```